### PR TITLE
grubby: add page

### PR DESCRIPTION
**Pull request by principis**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 8.40-75

### CODE REVIEW

**Comment by principis** [comment]
erm.. not sure how I request that review...? Sorry about that.

**Comment by kbdharun** [comment]
> erm.. not sure how I requested that review...? Sorry about that.

That's fine, Codeowners are automatically requested for reviews when the files they own are modified. Check out https://github.com/tldr-pages/tldr/blob/main/COMMUNITY-ROLES.md#codeowners for more information about it.

File: pages/linux/grubby.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# grubby
+
+> Command line tool for configuring grub and zipl.
```
Comment (kbdharun) [inline pages/linux/grubby.md:3]:
```suggestion
> Tool for configuring `grub` and `zipl`.
```
We generally don't use wording like "Command line tool" at the start and instead prefer just tool, etc. And we enclose commands inside backticks.

File: pages/linux/grubby.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://src.fedoraproject.org/rpms/grubby>.
```
Comment (kbdharun) [inline pages/linux/grubby.md:4]:
```suggestion
> More information: <https://manned.org/man/grubby.8>.
```
We generally prefer linking to the man page/usage documentation (if it is present then we reference the VCS repository.)

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by principis** [comment]
Thanks!

**Review by kbdharun — APPROVED** [review]

File: pages/linux/grubby.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# grubby
+
+> Tool for configuring `grub` and `zipl`.
```
Comment (acuteenvy) [inline pages/linux/grubby.md:3]:
```suggestion
> Tool for configuring `grub` and `zipl` bootloaders.
```

File: pages/linux/grubby.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://manned.org/man/grubby.8>.
+
+- Add kernel boot arguments to all kernel menu entries:
+
+`sudo grubby --update-kernel=ALL --args '{{quiet}} {{console=ttyS0}}'`
```
Comment (acuteenvy) [inline pages/linux/grubby.md:8]:
```suggestion
`sudo grubby --update-kernel=ALL --args '{{quiet console=ttyS0}}'`
```

**Review by acuteenvy — APPROVED** [review]

**Comment by principis** [comment]
Thanks for the suggestions!

**Review by kbdharun — APPROVED** [review]

**Pull request merged**